uber/templates/registration/new_base.html

```diff
@@ -147,7 +220,7 @@
         <td align="left">
             <a href="form?return_to=new%3f&id={{ attendee.id }}">{{ attendee.full_name }}</a>
             {% if attendee.banned %}
-                <div style="color:red">This attendee's name matches the name of a banned person.  Please get a department head to handle this.</div>
+                <div style="color:red">There is a problem with this registration. Please contact your department head.</div>
```

Comment:
I think it's still important to indicate *why* there is a problem with the registration. I'd suggest the following:
> This attendee's name matches the name of someone on the watchlist.  Please get a department head to handle this.